### ISSUE TITLE
Disabled Combobox label has insufficient color contrast

### ISSUE THREAD

**Issue by cnamal** [OfficeDev/office-ui-fabric-react#9909]
Go to https://developer.microsoft.com/en-us/fabric#/controls/web/combobox
Run [Accessibility insights](http://aka.ms/AccessibilityInsights)

-> A11y insights reports a color contrast issue
![image](https://user-images.githubusercontent.com/9625284/61700955-9ff8be00-ad3d-11e9-9541-29b3f89ae5f3.png)

-- A11y insights information
Title: WCAG 1.4.3: Elements must have sufficient color contrast (#ComboBox114-label)
Tags: Accessibility, WCAG 1.4.3, color-contrast

Issue: Elements must have sufficient color contrast (color-contrast: https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=msftAI)

Target application title: Home - Office UI Fabric
Target application url: https://developer.microsoft.com/en-us/fabric#/controls/web/combobox

Element path: #ComboBox114-label

Snippet: <label id="ComboBox114-label" for="ComboBox114-input" class="ms-Label root-185">Disabled ComboBox</label>

How to fix:
Fix any of the following:
  Element has insufficient color contrast of 2.63 (foreground color: #a19f9d, background color: #ffffff, font size: 10.5pt, font weight: bold). Expected contrast ratio of 4.5:1

Environment:
Chrome version 75.0.3770.142

====

This accessibility issue was found using Accessibility Insights for Web 2.7.1 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

**Comment by ecraig12345** [comment]
Thanks for reporting this. The text is `semanticColors.disabledText` which points to `palette.neutralTertiary` (`#a19f9d` by default). So I think this will be an issue for **all disabled text**, not just the ComboBox's label.

It seems odd that we'd have this combination as part of the default Fluent theme if it's not accessible. @Vitalius1 or @betrue-final-final do you know what the right fix is here, or am I missing something? The one thing I can think of offhand is that the text is bold, which IIRC changes the rules for contrast (but Accessibility Insights might not know about that?).

**Comment by Vitalius1** [comment]
I recently submitted a PR (#9746) that even further aligns other input controls to the same design. I believe the specs say that the disabled controls should not be forced to follow this rule and I remember  multiple discussions with @betrue-final-final about that.

**Comment by Vitalius1** [comment]
https://www.w3.org/WAI/GL/low-vision-a11y-tf/wiki/Contrast_(Minimum)#SC_Text for more context

**Comment by betrue-final-final** [comment]
Yes, this is not a bug. Disabled text doesn’t need to meet 4.5:1

**Comment by cnamal** [comment]
Opened an issue on Accessibility Insights. It's possible the issue is actually in the underlying axe-core engine. We'll see what their feedback is :)

**Comment by ecraig12345** [comment]
Thanks for doing that! Are you okay with closing the issue here?

**Comment by cnamal** [comment]
Sure!
